File: app/src/main/java/com/aline/teste/Models/Month.java
package com.aline.teste.Models;


import com.google.gson.annotations.SerializedName;


public class Month{

	@SerializedName("fund")
	private double fund;

	@SerializedName("CDI")
	private double cDI;

	public void setFund(double fund){
		this.fund = fund;
	}

	public double getFund(){
		return fund;
	}

	public void setCDI(double cDI){
		this.cDI = cDI;
	}

	public double getCDI(){
		return cDI;
	}
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/Month.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Month: Serializable {

    @SerializedName("fund")
    var fund: Double? = null
    @SerializedName("CDI")
    var cdi: Double? = null

    override fun toString(): String {
        return "Month(fund=$fund, cdi=$cdi)"
    }
}